docs/verify/simulate-with-qiskit-aer.ipynb

```diff
@@ -0,0 +1,255 @@
+    "    backend_options={\"noise_model\": noise_model}, approximation=True\n",
+    ")\n",
+    "job = noisy_estimator.run(circuit, observable, params)\n",
+    "noisy_value = job.result().values[0]\n",
+    "noisy_value"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As you can see, the expectation value in the presence of the noise is quite far from the correct value. In practice, you can employ a variety of error mitigation techniques to counter the effects of the noise, but a discussion of these techniques is outside the scope of this guide.\n",
+    "\n",
+    "To get a very rough sense of how the noise affects the final result, consider our noise model, which adds a depolarizing error of 2% to each CX gate. Depolarizing error with probability $p$ is defined as a quantum channel $E$ that has the following action on a density matrix $\\rho$:\n",
+    "\n",
+    "$$\n",
+    "E(\\rho) = (1 - p) \\rho + p\\frac{I}{2^n}\n",
+    "$$\n",
+    "\n",
+    "where $n$ is the number of qubits, in this case, 2. That is, with probability $p$, the state is replaced with the completely mixed state, and the state is preserved with probability $1 - p$. After $m$ applications of the depolarizing channel, the probability of the state being preserved would be $(1 - p)^m$. Therefore, we expect the probability of retaining the correct state at the end of the simulation to go down exponentially with the number of CX gates in our circuit.\n",
+    "\n",
+    "Let's count the number of CX gates in our circuit and compute $(1 - p)^m$. Because our circuit uses the EfficientSU2 class, we'll need to call `decompose` once to decompose it into CX gates. We call `count_ops` to get a dictionary that maps gate names to counts, and retrieve the entry for the CX gate."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.6542558123199923"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cx_count = circuit.decompose().count_ops()[\"cx\"]\n",
+    "(1 - cx_depolarizing_prob) ** cx_count"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This value, 65%, gives a rough estimate of the probability that our final state is correct. It is a conservative estimate because it does not take into account the initial state of the simulation. To get a more concrete estimate of how much our final state deviates from the correct state, let's use the [Sampler](https://qiskit.org/ecosystem/aer/stubs/qiskit_aer.primitives.Sampler.html) primitive to estimate the final measurement probability distributions with and without noise, and then compute the fidelity between these distributions. When running the Sampler, we pass `shots=None` to request that the final distribution not include any random sampling error."
```

Comment:
```suggestion
    "This value, 65%, gives a rough estimate of the probability that our final state is correct. It is a conservative estimate because it does not take into account the initial state of the simulation. To get a more concrete estimate of how much our final state deviates from the correct state, let's use the [Sampler](https://qiskit.org/ecosystem/aer/stubs/qiskit_aer.primitives.Sampler.html) primitive to estimate the final measurement probability distributions with and without noise, and then compute the fidelity between these distributions. When running the Sampler, we pass `shots=None` to request that the final distribution does not include any random sampling error."
```

Reply:
I actually prefer my wording here. Is it actually confusing or grammatically incorrect?

Reply:
I agree it's awkward, but IBM Style guide wants us not to use the subjunctive. How about 
"When running the Sampler, pass `shots=None` to exclude any random sampling error from the final distribution."

Reply:
I've rewritten it as "When running the Sampler, we pass `shots=None` to request a final distribution that does not include random sampling error."